pkg/build/ignore/docker/dockign.go

```diff
@@ -0,0 +1,116 @@
+package docker
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/openshift/source-to-image/pkg/api"
+)
+
+type DOCKIGN struct {
+	Config *api.Config
+}
+
+func New(req *api.Config) *DOCKIGN {
+	d := &DOCKIGN{
+		Config: req,
+	}
+	return d
+}
+
+func (b *DOCKIGN) Ignore(config *api.Config) {
+	path := filepath.Join(config.WorkingDir, "upload", "src", api.DefaultIgnoreFile)
+	file, err := os.Open(path)
+	if err != nil {
+		glog.V(4).Infof("Ignore processing, problem opening %s because of %v\n", path, err)
```

Comment:
need to break out this error handling...if the file doesn't exist, that's fine, but if something else went wrong we need to warn the user and abort the build, otherwise we're going to build something that includes files they did not intend.